{"name": "test_login", "status": "failed", "statusDetails": {"message": "AssertionError: assert <bound method User.get_user of <api.user.User object at 0x000001291BC64C50>> == 1", "trace": "get_user_1 = <bound method User.get_user of <api.user.User object at 0x000001291BC64C50>>\n\n    def test_login(get_user_1):\n        result = get_user_1\n        print(result)\n>       assert  result == 1\nE       assert <bound method User.get_user of <api.user.User object at 0x000001291BC64C50>> == 1\n\ntestcases\\test_01.py:6: AssertionError"}, "attachments": [{"name": "stdout", "source": "98e32929-e2f7-4420-880f-8fba316b2d59-attachment.txt", "type": "text/plain"}], "start": 1722509910345, "stop": 1722509910345, "uuid": "1cecc93a-15a8-483b-b3fc-49a4d67a7277", "historyId": "b9ce00cfe2210bb9aabb3ffe9d8d4d05", "testCaseId": "83fcd9480974c850c0e6a071e6bf7cd2", "fullName": "testcases.test_01#test_login", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_01"}, {"name": "host", "value": "DESKTOP-DVLO8G1"}, {"name": "thread", "value": "6032-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_01"}]}